spec/robots/dor_repo/was_crawl_dissemination/cdxj_merge_spec.rb

```diff
@@ -10,8 +10,8 @@
     let(:index_dir) { 'tmp/cdxj' }
     let(:working_dir) { "tmp/#{druid}" }
     let(:main_file) { "#{index_dir}/level0.cdxj" }
-    let(:working_file1) { "#{working_dir}/index1.cdxj" }
-    let(:working_file2) { "#{working_dir}/index2.cdxj" }
+    let(:working_file1) { "#{working_dir}/index1.cdxj" } # rubocop:disable RSpec/IndexedLet
+    let(:working_file2) { "#{working_dir}/index2.cdxj" } # rubocop:disable RSpec/IndexedLet
```

Comment:
i'd be fine with blanket disabling this rule, but also fine with inline exceptions as needed.